src/briefcase/console.py

```diff
@@ -161,13 +188,13 @@ def _log(
         :param style: Rich style to apply to everything printed for message.
         """
         if not message:
-            # When a message is not provided, do not output anything;
+            # When a message is not provided, only output the context;
             # This type of call is just clearing some vertical space.
-            self.print(show=show)
+            self.print(escape(self._context), show=show)
         else:
             if prefix:
                 # insert vertical space before for all messages with a prefix
-                self.print(show=show)
+                self.print(escape(self._context), show=show)
```

Comment:
I think this may be a slight over-correction.

Escaping content for Rich is only necessary when `markup=True`....otherwise, you risk inserting escape chars in the output.

This risk is mitigated here because we're already assuming that `context` will only be plain text....so escaping is basically a no-op.

However, this escaping is too lax to properly support control chars in `context` in some future state since it will escape whether `markup` is `True` or `False`. So, I think we'll be better off not escaping `context` at all for the time being.

(In retrospect, it may have been better to have always set `markup=True` to avoid this divergence in all logging.)

Reply:
Dang - that was a half thought that got accidentally committed. I came down on the side of fixing this with documentation, saying that context labels should be plain text, so the escape is unnecessary.